*** /C/WAMP/UniServer/www/drupal/sites/all/modules/civicrm/CRM/Contribute/Form/ContributionBase.php	Wed May 16 20:55:16 2012
--- ContributionBase.php	Tue Jun 12 12:09:52 2012
***************
*** 711,721 ****
--- 711,723 ----
  
        if ($fields) {
          // unset any email-* fields since we already collect it, CRM-2888
+         /* don't unset email fields for OSA because we don't collect it
          foreach (array_keys($fields) as $fieldName) {
            if (substr($fieldName, 0, 6) == 'email-') {
              unset($fields[$fieldName]);
            }
          }
+         */
  
          if (array_intersect_key($fields, $fieldsToIgnore)) {
            $fields = array_diff_key($fields, $fieldsToIgnore);
